cartridges/int_algolia_sfra/cartridge/static/default/js/lib/algolia/recommend-js.js

```diff
@@ -1,3 +1,2 @@
-//# sourceMappingURL=index.js.map
```

Comment:
This is a generated file, we shouldn't modify it. If there is an issue, better to report it to the team

Reply:
Did you confirm that it was causing a problem? Not being able to load a map file shouldn't cause any problem.

Reply:
What "approach"?
There is no "copy-file" logic, it's how it's released even on CDN: https://cdn.jsdelivr.net/npm/@algolia/recommend-js
You can ask the recommend team about it, but this is usually generated by minification tools, I don't believe there is anything to fix.

Reply:
As I said that it is only causing a console error and doesn't cause any functionality problem. Here is a screenshot.

![Image2024-04-19 at 21 38 39@2x](https://github.com/algolia/algoliasearch-sfcc-b2c/assets/36648230/0861c593-fe37-4300-8aab-fb3ec97cf38a)

I thought that it is not happening when we used it from CDN because https://cdn.jsdelivr.net/npm/@algolia/index.js.map accessible but it is also not accessible when we use CDN. So, I will let recommend team and update the PR